src/views/components/DisplayCard.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+	<div class="pet-card" :style="getCardStyle()">
+		<img class="xp-or-tier" v-if="isPet" :src="getXpPath()" />
+        <img class="xp-or-tier" v-if="isShop" :src="getTierPath()" />
+		<img class="pet-animal-img" :src="getPetImagePath()" />
+		<img class="pet-pedestal-img" src="../../assets/images/layout/shop-pedestal-with-shadow.png" />
+
+        <template v-if="!isShopFood">
+            <img class="pet-stats-img" src="../../assets/images/layout/animal-stats-template.png" />
+            <span class="card-text text-attack">{{ asset.attack }}</span>
+            <span class="card-text text-health">{{ asset.health }}</span>
+        </template>
+	</div>
+</template>
+
+<script>
+export default {
+    name: 'DisplayCard',
+    props: {
+        asset: {
+            type: Object,
+            required: true,
+        },
+        type: {
+            type: String,
+            required: true,
+            description: 'pet/shopPet/shopItem',
+        },
+    },
+    data:()=>({
+        cardWidth: 8,
+        cardHeight: 12,
+    }),
+    methods: {
+        getPetImagePath: function () {
+            let path = '';
+            if(this.isPet || this.isShopPet) {
+                path = '/src/assets/images/pets/' + this.asset.image + '.webp';
+            } else if (this.isShopFood) {
+                path = '/src/assets/images/items/' + this.asset.image + '.webp';
+            }
+            return path;
+        },
+        getCardStyle: function () {
+            return 'width:' + this.cardWidth + 'rem;height:' + this.cardHeight + 'rem';
+        },
+        getXpPath: function () {
+            return '/src/assets/images/misc/xp' + this.asset.xp + '.png';
+        },
+        getTierPath: function () {
+            return '/src/assets/images/misc/Tier_' + this.asset.tier + '_Icon.webp';
```

Comment:
I had renamed the tier icons to something like "iconTier1".

Were you on the latest code, or did I not push that, or was it all a fever dream?

Reply:
idk, i'm going to push what i have fixed from this stuff but i rebased on main and the only files that were renamed were the ones with underscores in them